tests/test_api.py

```diff
@@ -320,9 +321,9 @@ def test_if_in_assetview(self, request_headers, client, entity_id):
 @pytest.mark.schematic_api
 class TestMetadataModelOperation:
     @pytest.mark.parametrize("as_graph", [True, False])
-    def test_component_requirement(self, client, data_model_jsonld, as_graph):
+    def test_component_requirement(self, client, data_model_url, as_graph):
```

Comment:
This is neat @lakikowolfe , just for my education, does this mean that the tests run twice for both data_model_urls?

Reply:
Yep that's correct